08_Intro_Roman_Pipeline/08-Intro_Roman_Pipeline.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>should the console examples have <code>%</code> in front of them? It would be easier if the user could select the entire thing and copy / paste</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>